gatorgrouper/tests/test_read_student_file.py

```diff
@@ -0,0 +1,7 @@
+"""Testing number of groups"""
+from utils import read_student_file
+
+
+def test_read_file_populates_csvdata_0():
```

Comment:
Why does this method name end with `_0`?

Additionally, the `!= 0` does not actually check anything -- you should check that it contains something, perhaps by checking if it is truthy, or better yet, by constructing a `csv` file in code and then checking it reads the correct data.

This is a really bad test.

Reply:
We are going to rewrite this test function.